src/pages/hero/hero.scss

```diff
@@ -0,0 +1,100 @@
+@import "../../variables";
+
+.hero{
+    background-color: #F2F0FF;
```

Comment:
you're not using variables for colors))))))

Reply:
I didn't add it to the variables because I used it once) should I add it?

Reply:
yes, because same colors might be using in other components